client/src/containers/ResetPassword/ResetPassword.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
I think, this is not component, so stories for that is unnecessary

Reply:
Not necessary but i belive it is nice to have it